lstchain/calib/camera/calibrator.py

```diff
@@ -189,7 +189,7 @@ def _calibrate_dl1(self, event, telid):
         # TBD: - perform calibration of the R1 waveform (not DL1)
         #      - gain selection before charge integration
 
-
+        # In case of no gain selection the selected gain channels are  [0,0,..][1,1,..]
```

Comment:
This is not "no gain selection", right? This is "always choose high-gain"?

Reply:
no, this is no gain selection, why do you say that?